src/utils/seed.ts

```
@@ -0,0 +1,678 @@
+import { addressesSchema, backersSchema, categoriesSchema, organizationsSchema, paymentAccountsSchema, payoutSchedulesSchema, projectMediaSchema, projectsSchema, projectTagsSchema, publishersSchema, rewardsSchema, tagsSchema, taxInformationSchema, userFollowsSchema, usersSchema, type TNewCategory, type TNewOrganization, type TNewTag, type TNewUser } from '@/models/Schema';
+import { faker } from '@faker-js/faker';
+import { eq } from 'drizzle-orm';
+import { drizzle } from 'drizzle-orm/neon-http';
+import { neon } from '@neondatabase/serverless';
+import { config } from 'dotenv';
```

ESLint

[error] Run autofix to sort these imports!

(simple-import-sort/imports)